.env.demo.compose

```
@@ -565,3 +565,42 @@
+MCP_AUTH_RESOURCE_URI=https://mcpdemo.gauzy.co
+
+# JWT validation for demo (using RS256 with public key)
+MCP_AUTH_JWT_ALGORITHMS=RS256,ES256
+MCP_AUTH_JWT_AUDIENCE=https://mcpdemo.gauzy.co
+MCP_AUTH_JWT_ISSUER=https://authdemo.gauzy.co
+
+# Option 1: Use JWKS URI for dynamic key discovery (recommended)
+MCP_AUTH_JWT_JWKS_URI=https://authdemo.gauzy.co/.well-known/jwks.json
+
+# Demo authorization server configuration
+MCP_AUTH_SERVERS='[{
```

dotenv-linter (3.3.0)

[warning] [DuplicatedKey] The MCP_AUTH_SERVERS key is duplicated

(DuplicatedKey)